Add the ability to mute and unmute all players

kashike opened the pull request:
This commit adds the /unmute command, and removes the unmute ability from /mute.
Ticket: ESS-4941

prophile commented on Essentials/src/com/earth2me/essentials/utils/DateUtil.java:
```diff
@@ -70,7 +70,7 @@ public static long parseDateDiff(String time, boolean future) throws Exception
 		if (!found)
 		{
-			throw new Exception(_("illegalDate"));
+			throw new IllegalArgumentException(_("illegalDate"));
```
Why is this change here?

kashike replied:
In my opinion, throwing an Exception isn't very specific. I chose to use IllegalArgumentException because it kind of suits this, and I didn't want to create a new Exception class just for this.

Commit:
Add the ability to mute and unmute all players.
This commit adds the /unmute command, and removes the unmute ability from /mute.
Ticket: ESS-4941

khobbits commented:
Against this, we don't need more command bloat.
